gulp-tasks/reference-docs/get-source-code.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const fs = require('fs-extra');
+const wfHelper = require('../wfHelper');
+
+const getSourceCode = (gitUrl, tag, tmpPath) => {
```

Comment:
Please document functions with JSDoc. Include at least a brief description, and `@param`/`@return` as appropriate.  Will still merge if not added.